Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -545,6 +608,193 @@ namespace AzToolsFramework
+            {
+                m_columnLayout->RemoveSharePriorColumn(domIndex);
+                m_domOrderToAttributeInfo.erase(foundEntry);
+            }
+            // At least one attribute is valid, check if we need to update the existing attribute entry
+            else
+            {
+                // Check Alignment
+                if (alignment != foundEntry->second->m_alignment)
+                {
+                    foundEntry->second->m_alignment = alignment;
+                }
+
+                // Check SharePrior
+                if (sharePrior && !foundEntry->second->m_sharePriorColumn)
+                {
+                    AZ_Assert(priorColumnIndex != -1, "Tried to share column with an out of bounds index!")
+                    if (priorColumnIndex != -1)
+                    {
+                        m_columnLayout->AddSharePriorColumn(priorColumnIndex, domIndex);
+                        foundEntry->second->m_sharePriorColumn = true;
+                    }
+                }
+                else if (!sharePrior && foundEntry->second->m_sharePriorColumn)
+                {
+                    m_columnLayout->RemoveSharePriorColumn(domIndex);
+                    foundEntry->second->m_sharePriorColumn = false;
+                }
+
+                //Check MinimumWidth
+                if (minimumWidth != foundEntry->second->m_minimumWidth)
+                {
+                    foundEntry->second->m_minimumWidth = minimumWidth;
+                }
+
+                // Check DescriptionString
+                if (descriptionString.empty() && !foundEntry->second->m_descriptionString.empty())
+                {
+                    if (!toolTip().isEmpty())
+                    {
+                        setToolTip(QString());
+                    }
+                    if (!childWidget->toolTip().isEmpty())
+                    {
+                        childWidget->setToolTip(QString());
+                    }
+                    foundEntry->second->m_descriptionString = "";
+                }
+                else if (!descriptionString.empty() && foundEntry->second->m_descriptionString.empty())
+                {
+                    if (toolTip().isEmpty())
+                    {
+                        setToolTip(QString::fromUtf8(descriptionString.data(), aznumeric_cast<int>(descriptionString.size())));
+                    }
+                    if (childWidget->toolTip().isEmpty())
+                    {
+                        childWidget->setToolTip(QString::fromUtf8(descriptionString.data(), aznumeric_cast<int>(descriptionString.size())));
+                    }
+                    foundEntry->second->m_descriptionString = descriptionString;
+                }
+
+                // Check Disabled
+                if (shouldDisable != foundEntry->second->m_shouldDisable)
+                {
+                    foundEntry->second->m_shouldDisable = shouldDisable;
+                }
```

Comment:
I believe this needs an else case to cover when the Disabled attribute is removed.